WCON_format.md

```diff
@@ -129,23 +131,29 @@ A WCON file with arrayed t and corresponding nested arrays for `x` and `y`.
 }
 ```
 
-Note that even if some xy-coordinate arrays are nested, it is still possible to repeat an `id` in the same file if the same worm has more tracks later.  Allowing repeats in `id` is convenient for multi-worm tracking data as tracks can be joined simply by updating `id` without having to copy track data.
-
 #### Unknown Contents
 
 **Any software that reads WCON should ignore any unrecognised key-value pair**. This is essential to make WCON flexible and allows the inclusion of custom features and additional information not required by the basic format. Below we specify a method for including custom features and other additions that are included in the accompanying software examples.
 
 #### A Word about Numbers
 
-All numeric values may be floating-point.  However, JSON does not support infinite or not-a-number values permitted by IEEE754 standards that cover almost every floating-point number you might use on a computer.  Since WCON must be valid JSON, custom extensions to JSON (i.e. _not JSON_ but almost) like using `infinity` unquoted to denote a floating-point positive infinity, must be avoided.
+All numeric values may be floating-point.  However, JSON does not support infinite or not-a-number values permitted by the IEEE754 standards that are used by almost every programming language.  Since WCON must be valid JSON, custom extensions to JSON (i.e. _not JSON_ but almost) like using `infinity` unquoted to denote a floating-point positive infinity, must be avoided.
 
 If possible, the best solution is to simply leave out non-finite numbers.  However, in some cases (e.g. with arrayed `t`) it may be more practical to indicate missing data inline.  If you wish to embed not-a-number values, use `null`.  Avoid positive or negative infinity; this is not physical data anyway, so can just be replaced with `null`.
```

Comment:
We don't have any values where the distinction would be meaningful anyway, and in custom code you can do whatever you like.